### PR TITLE
Sign out via Magna

### PR DESCRIPTION
**Pull request by rafa-garcia**
To do (delete all that do not apply):

 - [ ] Changelog entry added.
 - [ ] (if there are vulnerable requirements) Upgraded any vulnerable dependencies.
 - [ ] (if updating requirements) Requirements have been compiled.
 - [ ] (if adding CSS) CSS have been compiled.
 - [ ] (if adding HTML) Change is accessible to the standards we subscribe to.
 - [ ] (if adding env vars) Added any new environment variable to vault.
 - [ ] (if adding feature flags) Cleaned up old flags
 - [ ] (if changing the UI) Add a printscreen to the PR

### CODE REVIEW

File: directory_components/templates/directory_components/header_footer/magna_header.html [hunk 1]
```diff
@@ -26,4 +26,27 @@
...
+          method: 'post',
+          headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
+            'X-CSRFToken': '{{ csrf_token }}',
+            'X-Requested-With': 'XMLHttpRequest',
+          },
+          body: JSON.stringify({})
+        }).then(function() {
+          window.location = '/';
```
Comment (bobmeredith) [inline directory_components/templates/directory_components/header_footer/magna_header.html:47]: I'm wondering where this is going. Isn't this the same as 'magna_home' above if all the webservers are running off the same domain. If they don't, we're dropping to the root of this server's domain.
For consistency, i'd suggest making this {{ header_footer_urls.magna_home }}

File: directory_components/templates/directory_components/header_footer/magna_header.html [hunk 2]
```diff
@@ -26,4 +26,27 @@
...
+
+    if (signOutButton) {
+      signOutButton.addEventListener('click', function(e) {
+        e.preventDefault();
+        fetch(sign_out_endpoint, {
+          method: 'post',
+          headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
+            'X-CSRFToken': '{{ csrf_token }}',
```
Comment (bobmeredith) [inline directory_components/templates/directory_components/header_footer/magna_header.html:42]: Hmm, not sure about this. Wouldn't this csrf token be generated by webserverx - whereas we are posting to the Magna WS so the tokens wouldn't match?

Comment (rafa-garcia) [reply directory_components/templates/directory_components/header_footer/magna_header.html:42]: Removing the csrf token from the header as the API endpoint for logging out does not allow cookie-based authentication

**Review by bobmeredith — APPROVED** [review]
LGTM

**Pull request merged**